frontend/src/store/modules/user.js

```diff
@@ -0,0 +1,63 @@
+import { sendRequest } from "@/javascript/utils/axiosUtils.js";
+
+export default {
+    namespaced: true,
+    state: {
+        loggedIn: false,
+        user: null,
+    },
+
+    getters: {
+        loggedIn(state) {
+            return state.loggedIn;
```

Comment:
Proponuję nie przechowywać tego pola loggedIn w store, tylko w tym getterze zwracać `state.user != null`. Wtedy ze store można usunąć pole `loggedIn` (i setter), a korzystać z tego właśnie przez ten getter. Głównie chodzi mi o to, żeby uniknąć sytuacji, że pole `loggedIn` będzie ustawione na true, ale nie będziemy mieć obiektu usera, wtedy może to być problematyczne bo nie wiadomo czy traktować go jako zalogowanego czy nie.